src/routes/tools/_common/_constants.js

```diff
@@ -0,0 +1,120 @@
+import models from "~/helpers/climate-models";
+import scenarios from "~/helpers/climate-scenarios";
+import boundaries from "~/helpers/mapbox-layers";
+
+export const DEFAULT_LOCATION = {
+  id: "37907",
+  title: "240 32nd Street, Sacramento, California 95816",
+  geometry: {
+    type: "Polygon",
+    coordinates: [
+      [
+        [-121.5, 38.625],
+        [-121.4375, 38.625],
+        [-121.4375, 38.5625],
+        [-121.5, 38.5625],
+        [-121.5, 38.625],
+      ],
+    ],
+  },
+  center: [-121.4688, 38.5938],
+  bbox: [-121.5, 38.5625, -121.4375, 38.625],
+};
+
+export const DEFAULT_SCENARIOS = scenarios.filter((d) =>
+  ["rcp45", "rcp85"].includes(d.id)
+);
+
+export const DEFAULT_BOUNDARIES = boundaries
+  .filter((d) => d.metadata.group === "Boundaries")
+  .map((d) => {
+    let label;
+    if (d.id === "locagrid") {
+      label = "None";
+    } else {
+      label = d.metadata.title;
+    }
+    return { ...d, label };
+  });
+
+export const SMALL_SCALE_BOUNDARIES = boundaries
+  .filter((d) =>
+    [
+      "locagrid",
+      "counties",
+      "censustracts",
+      "hydrounits",
+      "place",
+      "cdistricts",
+    ].includes(d.id)
+  )
+  .map((d) => {
+    let label;
+    if (d.id === "locagrid") {
+      label = "None";
+    } else {
+      label = d.metadata.title;
+    }
+    return { ...d, label };
+  });
+
+export const PRIORITY_10_MODELS = models.map((d) => {
+  let label;
+  if (d.text) {
+    label = `${d.id} (${d.text})`;
+  } else {
+    label = d.id;
+  }
+  return {
+    ...d,
+    label,
+  };
+});
+
+export const PRIORITY_4_MODELS = models
+  .filter((d) => d.priority)
+  .map((d) => {
+    let label;
+    if (d.text) {
+      label = `${d.id} (${d.text})`;
+    } else {
+      label = d.id;
+    }
+    return {
+      ...d,
+      label,
+    };
+  });
+
+export const ENSEMBLES = [
+  {
+    id: "rcp45_range",
+    label: "Modeled RCP 4.5 Range",
+    color: "rgba(218, 222, 225, 1)",
+    org: "Geospatial Innovation Facility",
+  },
+  {
+    id: "rcp85_range",
+    label: "Modeled RCP 8.5 Range",
+    color: "rgba(218, 222, 225, 1)",
+    org: "Geospatial Innovation Facility",
+  },
+];
+
+export const OBSERVED = [
+  {
+    id: "livneh",
+    label: "Observed",
+    color: "rgba(110, 110, 110, 1)",
+    org: "University of Colorado Boulder",
+  },
+];
+
+export const INITIAL_CONFIG = {
+  boundaryId: "locagrid",
+  scenarioId: "rcp45",
+  climvarId: "tasmax",
+  modelIds: "HadGEM2-ES,CNRM-CM5,CanESM2,MIROC5",
```

Comment:
Does it make sense to always have this as a string? Why not store it as an array of strings?

Reply:
I think my reasoning for these initial values to be strings is: I could deconstruct a bookmark/sharelink (i.e. `query` param in `preload`) and add initial values to stores more easily. But no reason it can't be an array

Reply:
Switched to using an array to store list of modes in https://github.com/berkeley-gif/caladapt-website-2021/commit/f25a28f802717be9af89af7c984816dba74dc427